core/src/com/unciv/logic/civilization/CivInfoStats.kt

```diff
@@ -101,7 +101,7 @@ class CivInfoStats(val civInfo: CivilizationInfo){
         val statMap = HashMap<String, Float>()
         statMap["Base happiness"] = civInfo.getDifficulty().baseHappiness.toFloat()
 
-        var happinessPerUniqueLuxury = 5f + civInfo.getDifficulty().extraHappinessPerLuxury
+        var happinessPerUniqueLuxury = 4f + civInfo.getDifficulty().extraHappinessPerLuxury
```

Comment:
Apparently this is a difficulty-dependent value - on Settler and Chieftain difficulties it's 5, on other difficulties it's 4.

The fact that you add many different things into a single pull request makes it very difficult for me to actually commit anything, since oncewe've agreed on one thing you've already added another... =/

Reply:
please recheck it, i test it in settler difficulty the happiness +6 when you have 1 luxury.
about your advice, i will do as you wish.